common/middleware/locals.js

```diff
@@ -16,6 +17,7 @@ module.exports = function setLocals(req, res, next) {
     USER: req.session.user,
     CURRENT_LOCATION: req.session.currentLocation,
     MOVES_URL: req.session.movesUrl || movesApp.mountpath,
+    ALLOCATION_URL: allocationsApp.mountpath,
```

Comment:
I don't think you necessarily need to copy this pattern here. The purpose of the moves value is so that it will remember any current day/location/filter you have when you go back rather than taking you always to "today".

Allocations sounds like it will sit outside of that style so you can just link to the allocations base path from the controller.